src/tests/helpers.js

```diff
@@ -46,6 +46,23 @@ export const mockNumberInput = {
   type: 'number',
 };
 
+export const mockNumberInputNegative = {
+  title: 'Tabs',
+  description: 'How many open tabs do you have?',
+  'x-jsf-presentation': {
+    inputType: 'number',
+  },
+  minimum: -100,
+  maximum: 0,
+  type: 'number',
+};
+
+export const schemaInputTypeNumberZeroMaximum = JSONSchemaBuilder()
```

Comment:
suggestion: `JSONSchemaBuilder` was something we created earlier in the development, but we've been removing it as it creates an extra layer of complexity. In this case, it could be simply:

```js
export const schemaInputTypeNumberZeroMaximum = {
  properties: {
    tabs: {
      title: 'Tabs',
      description: 'How many open tabs do you have?',
      'x-jsf-presentation': {
        inputType: 'number',
      },
      minimum: -100,
      maximum: 0,
      type: 'number',
    },
  },
};
```

P.S. We don't even need to import `mockNumberInputNegative` as it's not used anywhere else.